practice-one/src/components/food-card.html

```diff
@@ -0,0 +1,22 @@
+<div class="food-card-container">
+  <div class="food-card-wrapper">
+    <img src="{{image}}" alt="{{imageAlt}}" />
```

Comment:
No need to pass `imageAlt` use the `name` instead

Reply:
Thanks